### PR TITLE
Updating migrations page

### PR DESCRIPTION
**Pull request by jeffreylees**
https://auth0-docs-content-pr-5738.herokuapp.com/docs/migrations

### CODE REVIEW

File: articles/migrations/index.md [hunk 1]
```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
 
 If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
-### Introducing Resource Owner Support for oauth/token Endpoint
+---
+
+## 2018 Upcoming Endpoint Migrations
+
+::: note
```
Comment (mpaktiti) [inline articles/migrations/index.md]: I'd remove the panel here and leave the text as the introduction to this section (it's not really additional information)

File: articles/migrations/index.md [hunk 2]
```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
...
+Based on customer feedback, we have adjusted our plans and will continue to maintain and support the below listed proprietary endpoints. We will also publish guidance on the most effective ways to transition your applications to standards-based protocols. In the event that we need to make security enhancements to any of these legacy endpoints, we will promptly announce timeframes and guidelines for any required changes.
```
Comment (mpaktiti) [inline articles/migrations/index.md]: Maybe simplify the language a bit? For example `In the event that we need` can be `If we need`.

File: articles/migrations/index.md [hunk 3]
```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
...
 | --- | --- | --- |
-| Medium | 2017-12-21 |  2018-07-01 |
+| Low | TBD | TBD |
 
 Support was introduced for [Resource Owner Password](/api/authentication#resource-owner-password) to the [/oauth/token](/api/authentication#authorization-code) endpoint earlier this year. 
 
-We will now deprecate the current [/oauth/ro](/api/authentication#resource-owner) and [/oauth/access_token](/api/authentication#social-with-provider-s-access-token) endpoints.
-
-Applications must be updated before July 1, 2018, when [/oauth/ro](/api/authentication#resource-owner) will become unavailable. The migration guide will be available in Q1 2018.
+The current [/oauth/ro](/api/authentication#resource-owner) and [/oauth/access_token](/api/authentication#social-with-provider-s-access-token) endpoints will be deprecated at some point in the future.
```
Comment (mpaktiti) [inline articles/migrations/index.md]: `at some point in the future` or within 2018?

File: articles/migrations/index.md [hunk 4]
```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
...
+::: note
+Based on customer feedback, we have adjusted our plans and will continue to maintain and support the below listed proprietary endpoints. We will also publish guidance on the most effective ways to transition your applications to standards-based protocols. In the event that we need to make security enhancements to any of these legacy endpoints, we will promptly announce timeframes and guidelines for any required changes.
+:::
+
+### Resource Owner Support for oauth/token Endpoint
 
 | Severity | Grace Period Start | Mandatory Opt-In|
 | --- | --- | --- |
-| Medium | 2017-12-21 |  2018-07-01 |
+| Low | TBD | TBD |
```
Comment (mpaktiti) [inline articles/migrations/index.md]: Why did you change the severity to `Low`? Because it was pushed back or you were asked to do this specifically?

File: articles/migrations/index.md [hunk 5]
```diff
@@ -56,62 +45,68 @@ If you are currently implementing login in your application with Lock v8, v9, or
 
 If you have any questions, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
-### Introducing Resource Owner Support for oauth/token Endpoint
+---
+
+## 2018 Upcoming Endpoint Migrations
```
Comment (mpaktiti) [inline articles/migrations/index.md]: From the discussions I gathered that these migrations will start within 2018 (grace period start) and finish within 2018 (mandatory opt-in). Maybe the header should be `Upcoming Endpoint Migrations` and then clarify in the introductory text that the migrations will start within 2018 (as soon as we publish migration guides) and end (mandatory opt in) within 2019.

As is now I understand that these migrations will start and end within 2018.

File: articles/migrations/index.md [hunk 6]
```diff
@@ -21,25 +21,14 @@ During the grace period, customers are informed via dashboard notifications and
 
 If you need help with the migration, create a ticket in our [Support Center](${env.DOMAIN_URL_SUPPORT}).
 
+---
```
Comment (mpaktiti) [inline articles/migrations/index.md]: I'd remove the horizontal lines. This is not something we normally use, and if you think we should, we'd better follow up with design before start using them

**Review by mpaktiti — CHANGES_REQUESTED** [review]

File: articles/migrations/index.md (hunk 6 above)
Comment (jeffreylees) [reply articles/migrations/index.md]: I can't imagine what the problem with using an `hr` is, or why it would require design's input... they're perfectly normal demarcation characters... but I will remove them.
